src/Engine/ProtoCore/Core.cs

```diff
@@ -1510,17 +1510,20 @@ public void Cleanup()
 
         public void InitializeContextGlobals(Dictionary<string, object> context)
         {
-            int globalBlock = 0;
-            foreach (KeyValuePair<string, object> global in context)
+            if (context != null)
```

Comment:
The function is removed, AddContextData handles context injection generically for all types